### ISSUE TITLE
Is there any Visualization Examples here?

### ISSUE THREAD

**Issue by TwiceMao** [facebookresearch/theseus#646]
## ❓ Questions and Help
Is there any Visualization Examples here? I want to learn your libaray. But there is not visualazation examples.

**Comment by luisenp** [comment]
Hi @TwiceMao, could you be more specific on the type of visualization you are looking for?

**Comment by TwiceMao** [comment]
@luisenp For example, the change curve of the loss function. It would be best to have a comparison with traditional pytorch optimization.
